website/blog/2022-01-11-aluminium.md

```diff
@@ -0,0 +1,734 @@
+---
+author: Tomasz Godzik
+title: Metals v0.11.0 - Aluminium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.11.0! Due to the number of
+changes in this release we decided to introduce one more minor version before
+Metals 1.0.0, which is planned to come this year.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">336</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">136</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">11</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center"></td>
```

Comment:
I'm assuming this isn't supposed to be empty?

Reply:
I am adding the count now, good catch!

Reply:
Fixed